src/main/java/com/kodilla/ecommercee/controller/OrderNotFoundException.java

```diff
@@ -0,0 +1,4 @@
+package com.kodilla.ecommercee.controller;
+
+public class OrderNotFoundException extends Exception {
```

Comment:
błędy biznesowe powinny być dziedziczone po RuntimeException.
Może warto zastanowić się nad wprowadzeniem wspólnego mechanizmu obsługi błędów i mapowań błędów na odpowiednie responsy. 
Poczytaj:
https://www.baeldung.com/exception-handling-for-rest-with-spring 
https://mkyong.com/spring-boot/spring-rest-error-handling-example/
i porozmawiaj z kolegami czy będziecie się na to decydować

Reply:
zmienione